spec.go

```diff
@@ -83,6 +83,12 @@ var specCommand = cli.Command{
 						Destination: "/sys",
 						Options:     "nosuid,noexec,nodev",
 					},
+					{
+						Type:        "cgroup",
+						Source:      "cgroup",
+						Destination: "/sys/fs/cgroup",
+						Options:     "nosuid,noexec,nodev,relatime,ro",
```

Comment:
If `/sys/fs/cgroup` mounted as `ro`, we can't create subsystem dirs, I know it seems a bit strange since it looks different from what it shows.